packages/_helpers/locales/en/messages.po

```diff
@@ -16,5 +16,5 @@ msgstr ""
 #. Aria label for magnifying glass used as affix in Combobox and Textfield components
 #. js-lingui-explicit-id
 #: packages/_helpers/affix.tsx:37
-msgid "forms.affix.aria.magnifyingGlass"
-msgstr "Magnifying glass"
+#~ msgid "forms.affix.aria.magnifyingGlass"
+#~ msgstr "Magnifying glass"
```

Comment:
I wonder if this change makes anything work differently 🤔

Reply:
I have no idea, this just updated automatically when I ran `pnpm messages:extract` 🙈

Reply:
We can see later if that breaks anything, if now you’re unable to test it due to missing translations 🙂